src/distar.ml

```diff
@@ -0,0 +1,9 @@
+(**************************************************************************)
+(*  Copyright (C) 2018 Yann Régis-Gianas, Étienne Marais                  *)
+(*                                                                        *)
+(*  This is free software: you can redistribute it and/or modify it       *)
+(*  under the terms of the GNU General Public License, version 3.         *)
+(*                                                                        *)
+(*  Additional terms apply, due to the reproduction of portions of        *)
```

Comment:
This is imported from morbig, right? Be careful, the second part of the header is not relevant here since we will note cite the POSIX standard.